benchmarks/helics/BenchmarkFederate.hpp

```diff
@@ -0,0 +1,334 @@
+/*
+Copyright (c) 2017-2020,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC.  See
+the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
```

Comment:
Is there are reason we aren't using #pragma once here,  we have been moving away from include guards?